### ISSUE TITLE
No changes made on task laravel/unguarded-models, warning thrown 

### ISSUE THREAD

**Issue by ordago** [laravel-shift/workbench-issues#17]
**Desktop:**
 - OS: Windows
 - OS Version: 10
 - App Version: 0.3.2.0 - 0.4.0.0

**Describe the bug**
The log shows this:

```
cp: can't stat '/opt/shift/scripts/find-fillable-models.php': No such file or directory
rm: can't remove 'find-fillable-models.php': No such file or directory
```

**To Reproduce**
1. open workbench
2. select project
3. Run task

![Workbench_sLw5uvOAAR](https://user-images.githubusercontent.com/6376814/125471885-7cd17901-2e9e-41cf-ae41-7eace643e797.png)

`$fillable`s arrays in the code:

![zMj9vnf44p](https://user-images.githubusercontent.com/6376814/125471926-07334f8a-82fe-46d2-b885-fc28e75a1862.png)

**Comment by jasonmccreary** [comment]
Please update to the latest version (0.4.0) and let me know if the problem persists.

**Comment by ordago** [comment]
![image](https://user-images.githubusercontent.com/6376814/125485830-78d40628-e41b-4c4c-b538-51ab6b32b22b.png)


Same result as the previous version

![image](https://user-images.githubusercontent.com/6376814/125485702-79759599-a9c3-4939-b975-a2891fd0b073.png)

**Comment by jasonmccreary** [comment]
We'll push a patch for this shortly.

**Comment by jasonmccreary** [comment]
@ordago, this should be patched in the latest release (0.4.1). Again, there may still be issues with the automation. But the errors reported in this issue should be resolved.

**Comment by ordago** [comment]
It worked now. Thanks! 🎉
